{"source": "exploit-database", "filename": "platforms/linux/remote/40920.py", "cves": ["CVE-2016-9566", "CVE-2016-9565"]}